package.json

```diff
@@ -88,6 +91,12 @@
         "NODE_ENV": "test",
         "DEBUG": "app:*"
       }
+    },
+    "test:configs": {
+      "command": "rimraf tests/config/generated/tests.specs.json && rimraf tests/config/generated/tests.features.json && json-concat tests/config/specs.config.json tests/config/common.config.json tests/config/generated/tests.specs.json && json-concat tests/config/features.config.json tests/config/common.config.json tests/config/generated/tests.features.json",
```

Comment:
when i tried that, generating the json configs didn't work because concat-json won't create the dir if it doesn't exist - it just fails.